src/SFML/Window/Win32/WindowImplWin32.cpp

```diff
@@ -496,8 +496,8 @@ void WindowImplWin32::switchToFullscreen(const VideoMode& mode)
     }
 
     // Make the window flags compatible with fullscreen mode
-    SetWindowLongW(m_handle, GWL_STYLE, WS_POPUP | WS_CLIPCHILDREN | WS_CLIPSIBLINGS);
-    SetWindowLongW(m_handle, GWL_EXSTYLE, WS_EX_APPWINDOW);
+    SetWindowLongPtrW(m_handle, GWL_STYLE, WS_POPUP | WS_CLIPCHILDREN | WS_CLIPSIBLINGS);
+    SetWindowLongPtrW(m_handle, GWL_EXSTYLE, WS_EX_APPWINDOW);
```

Comment:
Only saw right now, it should be `SetWindowLongPtr` not `SetWindowLongPtrW`

Reply:
@eXpl0it3r, check it https://github.com/SFML/SFML/commit/57150e3d05def23ae1d8943d32beb7a29dfc350f